content/blog/overview-of-the-platform-level-data-model-for-redfish®-device-enablement-standard.md

```diff
@@ -0,0 +1,158 @@
+
+This statement is clearly mentioned in the `readme.txt` file embedded in the `ilo5_265.fwpkg` firmware package. You can use the `unzip` utility to extract this file from the package as shown in the next picture.
+
+![RDE support in iLO 5 firmware version 2.65](/img/rdesupportinfw265.png "RDE support in iLO 5 firmware version 2.65")
+
+
+
+### Storage controllers
+
+The [Enhancements paragraph](https://support.hpe.com/connect/s/softwaredetails?language=en_US&softwareId=MTX_a06269a339da44ada286007600&tab=Enhancements) of HPE Smart Arrays version 5.00 mentions several new PLDM and DMTF features for this version (see picture below).
+
+![PLDM Enhancements](/img/firmwareenhancements.png "PLDM Enhancements")
+
+
+
+If you power on a server containing this type of storage controller and the correct firmware versions, you will notice a device with string `DE` in its URI similar to : `/redfish/v1/systems/1/DExxxxxx`. This `DE` string is synonym of a device supporting the PLDM for RDE standard and discovered as such by the Redfish service.
+
+A GET request toward this URI returns links to controllers, drives and volumes property end points as shown in the next picture.
+
+[](RdeStorageEndPoint.png)
+
+![RDE storage end point](/img/rdestorageendpoint.png "RDE storage end point")
+
+
+
+If you want to create a Logical Drive, you can POST a request toward `{{iloURI}}/redfish/v1/Systems/1/Storage/DE07C000/Volumes` with a workload similar to:
+
+```json
+{
+    "RAIDType": "RAID1",
+    "DisplayName": "Raid1-1",
+    "ReadCachePolicy": "ReadAhead",
+    "VolumeUsage": "Data",
+    "InitializeMethod": "Background",
+    "Links": {
+        "Drives": [
+            {
+                "@odata.id": "/redfish/v1/Systems/1/Storage/DE07C000/Drives/0"
+            },
+            {
+                "@odata.id": "/redfish/v1/Systems/1/Storage/DE07C000/Drives/1"
+            }
+        ]
+    }
+}
+```
+
+The request will be transferred to the device and analyzed. If the device can process it, you will receive a `201 Created` HTTP response code. The just created Logical Drive can be used immediately by the operating system without any system reboot. You may have to ask the OS to re-scan storage devices to discover this newly created LD.
+
+The deletion of a RDE Logical Drive is straight forward; you just have to send a DELETE request toward the LD URI:
+
+```Bash
+curl --request DELETE "$iloURI/redfish/v1/Systems/1/Storage/DE07C000/Volumes/1"
+```
+
+**NOTE**: For backward compatibility and concerning HPE Smart Array SR devices, HPE Gen10 and Gen10 Plus servers offer the possibility to manage those devices using both `SmartStorageConfig` and `Storage` data types. However, always stick to a single management data type. Mixing those data types may lead to hazardous results.
```

Comment:
The first sentence is a little confusing. If you remove "and", I think it works... but that's your call. I think what you mean is "For backward compatibility concerning HPE Smart Array SR devices..."